api4/post_test.go

```diff
@@ -2963,6 +2963,34 @@ func TestGetPostsByIds(t *testing.T) {
 	CheckNotFoundStatus(t, response)
 }
 
+func TestGetEditHistoryForPost(t *testing.T) {
+	th := Setup(t).InitBasic()
+	defer th.TearDown()
+	client := th.Client
```

Comment:
Related question with above, if we need a permission check it's better to add cases for permissions. Also a test case if there is no post edits etc.

Reply:
Another good catch on the if trhere is no post edits